seqr/views/utils/file_utils.py

```diff
@@ -110,16 +117,27 @@ def save_uploaded_file(request, process_records=None, allow_json=False):
 
     # save json to temporary file
     uploaded_file_id = hashlib.md5(str(json_records).encode('utf-8')).hexdigest() # nosec
-    serialized_file_path = _compute_serialized_file_path(uploaded_file_id)
+    file_name = _compute_serialized_file_name(uploaded_file_id)
+    serialized_file_path = get_temp_file_path(file_name, is_local=True)
     with gzip.open(serialized_file_path, 'wt') as f:
         json.dump(json_records, f)
 
+    persist_temp_file(file_name, request.user)
+
     return uploaded_file_id, filename, json_records
 
 
-def load_uploaded_file(upload_file_id):
-    serialized_file_path = _compute_serialized_file_path(upload_file_id)
-    with gzip.open(serialized_file_path, "rt") as f:
-        json_records = json.load(f)
+def persist_temp_file(file_name, user, is_directory=False):
+    if not anvil_enabled():
```

Comment:
just to clarify in my mind - are there local users who have anvil enabled? In that case, we are allowing them to store uploads in our seqr-scratch-temp?